manifest-list.md

```diff
@@ -108,5 +116,5 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 }
 ```
 
-[runtime-platform2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md#platform
+[runtime-platform2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/config.md#platform
```

Comment:
If we bump this to rc3, we probably want to update the [config reference as well][1].  But we can probably leave the rc2 → rc3 bump out of this PR.

[1]: https://github.com/opencontainers/image-spec/blame/eee8749ab8dd46b26778f6d97a9d59e60aefe5d0/config.md#L232